### DISCUSSION TITLE
Using Da Vinci Resolve Speed Editor search dial to scrub in a standalone video player?

### DISCUSSION THREAD

**Discussion by victorhooi** [CommandPost/CommandPost#3349] [Technical Support]
I have a Da Vinci Resolve Speed Editor, and I would love to be able to use the search dial (aka shuttle dial, or jog dial) to scrub through video in a video player (e.g. mpv), or anything similar.

Has anybody tried something like this?

Accordiong to [this video](https://www.youtube.com/watch?v=M9KBrV65pfw), that person claims the jog dial doesn't work, as it times out after 5 minutes. Is this still the case, or is there some kind of workaround?

What settings did you use? ([This issue](https://github.com/CommandPost/CommandPost/issues/3292) mentions it's tricky to get it to mimic the native feel of the Speed Editor?)

**Comment by strykedits** [comment]
I am the creator of that video.  I'm posting a new video today that fixed the jog wheel problem.   Essentially you have to map a key to go to "bank1" of the Speed Editor.  It resets it.   It's another long video, but should help a lot of people out.

**Reply by strykedits** [reply]
Here is the video:  https://www.youtube.com/watch?v=PcyfasCqSJU
